src/uuid/mod.rs

```diff
@@ -34,41 +35,46 @@
 //!     bson_uuid: bson::Uuid,
 //!
 //!     /// serializes as a BSON binary with subtype 4 when either is used.
-//!     /// this requires the "uuid-0_8" feature flag
-//!     #[serde(with = "bson::serde_helpers::uuid_as_binary")]
+//!     /// this requires the "uuid-1" feature flag
+//!     #[serde(with = "bson::serde_helpers::uuid_1_as_binary")]
 //!     uuid_as_bson: uuid::Uuid,
 //! }
 //! # };
 //! ```
 //!
-//! ## The `uuid-0_8` feature flag
+//! ## The `uuid-1` feature flag
 //!
 //! To facilitate the conversion between [`crate::Uuid`] values and the `uuid` crate's `Uuid`
-//! values, the `uuid-0_8` feature flag can be enabled. This flag exposes a number of convenient
-//! conversions, including the [`crate::Uuid::to_uuid_0_8`] method and the `From<uuid::Uuid>`
+//! values, the `uuid-1` feature flag can be enabled. This flag exposes a number of convenient
+//! conversions, including the `crate::Uuid::to_uuid_1` method and the `From<uuid::Uuid>`
 //! implementation for `Bson`, which allows the `uuid` crate's `Uuid` values to be used in the
 //! `doc!` and `bson!` macros.
 //!
 //! ```
-//! # #[cfg(feature = "uuid-0_8")]
+//! # #[cfg(feature = "uuid-1")]
 //! # {
+//! # use uuid as uuid;
 //! use bson::doc;
 //!
 //! // this automatic conversion does not require any feature flags
 //! let query = doc! {
 //!     "uuid": bson::Uuid::new(),
 //! };
 //!
-//! // but this automatic conversion requires the "uuid-0_8" feature flag
+//! // but this automatic conversion requires the "uuid-1" feature flag
 //! let query = doc! {
 //!     "uuid": uuid::Uuid::new_v4(),
 //! };
 //!
-//! // this also requires the "uuid-0_8" feature flag.
-//! let uuid = bson::Uuid::new().to_uuid_0_8();
+//! // this also requires the "uuid-1" feature flag.
+//! let uuid = bson::Uuid::new().to_uuid_1();
 //! # };
 //! ```
 //!
+//! For backwards compatibility, a `uuid-0_8` feature flag can be enabled, which provides the same
+//! API for interoperation with version 0.8 of the `uuid` crate.  Note that at most one of `uuid-1`
```

Comment:
in a comment above @patrickfreed mentioned that the flags are not mutually exclusive, but this seems to imply they are -- which is accurate?

Reply:
Whoops, this is left over from when they were exclusive.  Fixed.